workspaces/airbrake/plugins/airbrake-backend/src/service/router.ts

```diff
@@ -91,6 +90,5 @@ export async function createRouter(
     }),
   );
 
-  router.use(errorHandler());
```

Comment:
this should become `middleware.error()` right?

Reply:
So, to be honest I'm not sure. We don't include that anymore in the template when creating a new backend plugin:

https://github.com/backstage/backstage/blob/994506952c20d7708c2f8a5f6498b19132157166/packages/cli/templates/default-backend-plugin/src/router.ts#L50

Reply:
Still needed to support legacy backend, will add that in 👍